src/SFML.Window/Clipboard.cs

```diff
@@ -29,10 +29,10 @@ public static string Contents
                     }
                 }
                 
-                byte[] sourceBytes = new byte[length * 4];
-                Marshal.Copy(source, sourceBytes, 0, sourceBytes.Length);
-                
-                return Encoding.UTF32.GetString(sourceBytes);
+                unsafe
+                {
+                    return Encoding.UTF32.GetString((byte*)source, (int)( length * 4 ));

```

Comment:
While I know that UTF32 is going to yield 4 bytes per char, does it make more sense to use `sizeof()` for clarity on this and the other uses of `UTF*.GetString()`?